voxblox/include/voxblox/integrator/tsdf_integrator_fast.h

```diff
@@ -77,6 +77,17 @@ class TsdfIntegrator {
     return 0.0;
   }
 
+  inline float getVoxelWeight(const Point& point_C) const {
```

Comment:
actually no, helen reminded me that the FloatingPoint typedef was originally  used to determine the precision of Points in space, not the signed distance function value